pkg/collector/reconcile/configmap_test.go

```diff
@@ -94,23 +94,11 @@ receivers:
       grpc: null
   prometheus:
     config:
-      global:
-        scrape_interval: 1m
-        scrape_timeout: 10s
-        evaluation_interval: 1m
       scrape_configs:
-      - job_name: otel-collector
-        honor_timestamps: true
+      - http_sd_configs:
+        - url: http://test-targetallocator:80/jobs/otel-collector/targets?collector_id=$POD_NAME
+        job_name: otel-collector
         scrape_interval: 10s
-        scrape_timeout: 10s
-        metrics_path: /metrics
-        scheme: http
-        follow_redirects: true
-        enable_http2: true
-        http_sd_configs:
-        - follow_redirects: false
-          enable_http2: false
-          url: http://test-targetallocator:80/jobs/otel-collector/targets?collector_id=$POD_NAME
```

Comment:
Why are we removing these lines?

Reply:
@jaronoff97 These are additional global defaults that are included in the prometheus configuration when it is unmarshalled. Since we are no longer unmarshalling the configuration, these defaults had to be removed. The [original configuration](https://github.com/open-telemetry/opentelemetry-operator/blob/main/pkg/collector/reconcile/test.yaml) does not contain these defaults.